scripts/DeployAndAddTracer.js

```diff
@@ -1,67 +1,54 @@
-const hre = require("hardhat")
-const TestToken = artifacts.require("TestToken")
-const { BN, time } = require("@openzeppelin/test-helpers")
-const Gov = artifacts.require("Gov")
-const TracerFactory = artifacts.require("TracerPerpetualsFactory")
+const hre = require("hardhat");
+const { BN } = require("@openzeppelin/test-helpers");
 
 // small sample script for using deploys and then deploying a trace
 async function main() {
     // deploy all contracts using hardhat deploy
-    const { deployments } = hre
-    const twoDays = 172800
-    let accounts = await web3.eth.getAccounts()
-
+    const { deployments } = hre;
+    const twoDays = 172800;
     // deploy all contracts
-    await deployments.fixture(["FullDeploy"])
-
-    // todo how do collisions work on  multiple instances of a contract?
-    let oracle = await deployments.get("Oracle")
-    let gasPriceOracle = await deployments.get("Oracle")
-    let account = await deployments.get("Account")
-    let pricing = await deployments.get("Pricing")
-    let maxLeverage = new BN("125000").toString()
-    let gov = await deployments.get("Gov")
-    let govToken = await deployments.get("TestToken")
-    let factory = await deployments.get("TracerFactory")
-    gov = await Gov.at(gov.address)
-    govToken = await TestToken.at(govToken.address)
-    factory = await TracerFactory.at(factory.address)
+    await deployments.fixture(["FullDeploy"]);
 
-    // get the deployed gov token and send it out
-    for (var i = 0; i < 6; i++) {
-        await govToken.transfer(accounts[i], web3.utils.toWei("200"))
-    }
+    let maxLeverage = new BN("125000").toString();
+    let tokenDecimals = new BN("1000000").toString();
+    let fundingRateSensitivity = 1;
+    let feeRate = 1;
+    let gasPriceOracle = await deployments.get('Oracle');
+    // let liquidation = await deployments.get('Liquidation');
+    let trader = await deployments.get("Trader");
+    let factory = await deployments.get("TracerPerpetualsFactory");
+    let token = await deployments.get("TestToken");
 
-    // use gov to deploy a tracer
-    var token = await TestToken.new(web3.utils.toWei("1000000"))
 
     //Deploy a new Tracer contract per test
-    var deployTracerData = web3.eth.abi.encodeParameters(
-        [
-            "bytes32",
-            "address",
-            "address",
-            "address",
-            "address",
-            "address",
-            "int256",
-            "uint256",
-        ],
-        [
-            web3.utils.fromAscii(`TEST1/USD`),
-            token.address,
-            oracle.address,
-            gasPriceOracle.address,
-            account.address,
-            pricing.address,
-            maxLeverage,
-            1, //funding rate sensitivity
-        ]
-    )
-
-    await factory.deployTracer(deployTracerData)
-    let tracerAddr = await factory.tracersByIndex(0)
-    console.log(`Tracer Deplpoyed: ${tracerAddr}`)
+    // var deployTracerData = ethers.utils.defaultAbiCoder.encode(
```

Comment:
If these are commented out, should we take this opportunity to simplify our deployment code now too? @raymogg